spec/concurrent/set_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'set' 
+module Concurrent
+  describe Set do 
+    let!(:set) { described_class.new }
+
+    it 'concurrency' do
+      (1..THREADS).map do |i|
+        Thread.new do
+           1000.times do |j|
+            set << i 
+            set.empty?
```

Comment:
This spec does not make set any expectations. Can you clarify what you are testing and make expectations about the behavior in the spec?

Reply:
@ianks actually this is my first ever contribution so i dont have much knowledge about minitests so can u plese help me out in this?

Reply:
Actually the tests follow what we have for Array and Hash. I'll improve all of them at least a bit.